File: calico-us-berkeley/2025-fall/final/reactions/templates/reactions.py
def solve(A: int, P: list[float]) -> None:
    """

    P: list of floating point numbers. Each floating point number represents 
    the probability between 0 and 1 of a successful reaction.

    Prints a sequence of actions to standard output that represents the best
    strategy to maximize the number of successful reactions.
    """

    # YOUR CODE HERE
    print("NEXT")

def main():
    P = []
    A = int(input())
    for _ in range(A):
        P.append(float(input()))
    solve(A, P)

if __name__ == '__main__':
    main()

File: calico-us-berkeley/2025-fall/final/reactions/templates/reactions.java
import java.io.*;
import java.util.ArrayList;

class Solution {
    /**
     *  P: list of floating point numbers. Each floating point number represents 
     *	the probability between 0 and 1 of a successful reaction.

     *	Prints a sequence of actions to standard output that represents the best
     *	strategy to maximize the number of successful reactions.
     */
	
	static BufferedReader in = new BufferedReader(new InputStreamReader(System.in));
    static PrintWriter out = new PrintWriter(System.out);
	
    static void solve(int A, ArrayList<Double> P) {
        // YOUR CODE HERE
    	out.println("NEXT");
    }
    

    public static void main(String[] args) throws IOException {
        int A = Integer.parseInt(in.readLine());
        ArrayList<Double> P = new ArrayList<>();
        for (int i = 0; i < A; i++) {
            String[] temp = in.readLine().split(" ");
            P.add(Double.parseDouble(temp[0])); 
        }
        solve(A, P);
        out.flush();
    }
}
